pom.xml

```diff
@@ -354,6 +353,12 @@
             <url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
         </snapshotRepository>
     </distributionManagement>
+
+    <modules>
+        <module>component/authenticator</module>
+        <module>feature/org.wso2.carbon.identity.authenticator.mobileconnect.server.feature</module>
+    </modules>
+
     <properties>
         <carbon.identity.version>5.0.7</carbon.identity.version>
```

Comment:
Where is this property used?
We don't use carbon.identity.version (this was used when all extension repos resided in carbon-identity repo) now since we have extension repos that are released independently.